setup.py

```diff
@@ -640,18 +646,31 @@ def determine_gl_flags():
         flags['library_dirs'] = [join(ndkplatform, 'usr', 'lib')]
         flags['libraries'] = ['GLESv2']
     elif platform == 'rpi':
-        flags['include_dirs'] = [
-            '/opt/vc/include',
-            '/opt/vc/include/interface/vcos/pthreads',
-            '/opt/vc/include/interface/vmcs_host/linux']
-        flags['library_dirs'] = ['/opt/vc/lib']
-        brcm_lib_files = (
-            '/opt/vc/lib/libbrcmEGL.so',
-            '/opt/vc/lib/libbrcmGLESv2.so')
+
+        if not cross_sysroot:
+            flags['include_dirs'] = [
+                '/opt/vc/include',
+                '/opt/vc/include/interface/vcos/pthreads',
+                '/opt/vc/include/interface/vmcs_host/linux']
+            flags['library_dirs'] = ['/opt/vc/lib']
+            brcm_lib_files = (
+                '/opt/vc/lib/libbrcmEGL.so',
+                '/opt/vc/lib/libbrcmGLESv2.so')
+
+        else:
+            print("KIVY_CROSS_SYSROOT: " + cross_sysroot)
+            flags['include_dirs'] = [
+                cross_sysroot + '/usr/include',
+                cross_sysroot + '/usr/include/interface/vcos/pthreads',
+                cross_sysroot + '/usr/include/interface/vmcs_host/linux']
+            flags['library_dirs'] = [cross_sysroot + '/usr/lib']
+            brcm_lib_files = (
+                cross_sysroot + '/usr/lib/libbrcmEGL.so',
+                cross_sysroot + '/usr/lib/libbrcmGLESv2.so')
+
         if all((exists(lib) for lib in brcm_lib_files)):
-            print(
-                'Found brcmEGL and brcmGLES library files'
-                'for rpi platform at /opt/vc/lib/')
+            print('Found brcmEGL and brcmGLES library files'
+                  'for rpi platform at ' + dirname(brcm_lib_files[0]))
```

Comment:
One space character is missing at then end of first part of the string, so this now prints "... library filesfor rpi..." instead of "... library files for rpi..."

Reply:
PR ? :)